web/app/scripts/custom-reports/aggregations-config.js

```diff
@@ -9,58 +9,62 @@
      */
 
     /* ngInject */
-    function AggregationsConfig($q, RecordState, RecordSchemaState, GeographyState) {
+    function AggregationsConfig($translate, $q, RecordState, RecordSchemaState, GeographyState) {
+        var timeType = $translate.instant('AGG.TIME');
```

Comment:
Is this ever user-visible? I might be mis-remembering but I thought this was passed to the API as a parameter for how to render the report. If that's the case it probably doesn't need to be translated, but if that's not right then feel free to ignore.